src/lib.rs

```diff
@@ -344,6 +289,22 @@ fn run<A, B, S>(backend: shade::Backend,
     }
 }
 
+trait EventsLoop {
```

Comment:
`winit` and `glutin` both implement the same methods but don't share a trait. Therefore, I created an own for our specific needs..